applications/osb-portal/src/components/workspace/WorkspaceEditor.tsx

```diff
@@ -107,69 +109,76 @@ async function readFile(file: Blob) {
     fileReader.onerror = reject;
 
     fileReader.readAsArrayBuffer(file);
-  })
+  });
 }
 
 let thumbnail: Blob;
 
 export default (props: WorkspaceEditProps) => {
-
   const classes = useStyles();
 
   const { workspace } = props;
-  const [workspaceForm, setWorkspaceForm] = React.useState<
-    Workspace
-  >({ ...props.workspace });
+  const [workspaceForm, setWorkspaceForm] = React.useState<Workspace>({
+    ...props.workspace,
+  });
 
   const closeWorkSpaceEditor = () => {
     if (props.closeHandler) {
       props.closeHandler();
     }
-  }
+  };
 
-  const [thumbnailPreview, setThumbnailPreview] = React.useState<any>(workspace?.thumbnail ? "/proxy/workspaces/" + workspace.thumbnail : null);
+  const [thumbnailPreview, setThumbnailPreview] = React.useState<any>(
+    workspace?.thumbnail ? "/proxy/workspaces/" + workspace.thumbnail : null
+  );
   const [thumbnailError, setThumbnailError] = React.useState<any>(null);
-  const [showNoFilesSelectedDialog, setShowNoFilesSelectedDialog] = React.useState(false);
-  const workspaceTags = workspace && workspace.tags ? workspace.tags.map((tagObject) => tagObject.tag) : [];
+  const [showNoFilesSelectedDialog, setShowNoFilesSelectedDialog] =
+    React.useState(false);
+  const workspaceTags =
+    workspace && workspace.tags
+      ? workspace.tags.map((tagObject) => tagObject.tag)
+      : [];
   const [defaultTags, setDefaultTags] = React.useState(workspaceTags);
 
   const handleCreateWorkspaceButtonClick = () => {
-    if (typeof props.filesSelected !== 'undefined') {
-      props.filesSelected ? handleCreateWorkspace() :
-        setShowNoFilesSelectedDialog(!showNoFilesSelectedDialog);
-    }
-    else {
+    if (typeof props.filesSelected !== "undefined") {
+      props.filesSelected
+        ? handleCreateWorkspace()
```

Comment:
Need to set up the linter not to change previously existing lines. Only real changes shouls pop up